src/Instantiator/ObjectInstantiator.php

```diff
@@ -14,6 +14,10 @@
 
 namespace SensioLabs\RichModelForms\Instantiator;
 
+use Closure;
```

Comment:
Can you revert the `use` statements for classes from the global namespace please? :)

Reply:
@xabbuh you should probably force this at `php-cs-fixer` level.